content/updates/2020-10/3-Better-manage-product-models-and-variants-in-the-grid.md

```diff
@@ -1,26 +1,49 @@
 ---
-pim_announcement_alt_img: ""
+pim_announcement_img: ./img/better-manage-product-models-and-variants-in-the-grid.png
+pim_announcement_alt_img: "New counters and display option in the product grid"
 pim_announcement_audience:
 - CE
 - EE
 ---
 
 # Better manage product models and variants in the grid
 
-TODO
+Do you work a lot with product models and variants? If yes, you will surely enjoy these simple but yet efficient improvements we made in the product grid: a new filter on the product type, a new display option...
+
 ## Filter on the product typology
-::: meta-data type="New" features="Productivity" available="early October" link-to-doc="../articles/.html"
+::: meta-data type="New" features="Productivity" available="early October" link-to-doc="../articles/enrich-products-variants.html#filter-on-product-typology"
 :::
 
+The new `Product typology` filter in the product grid allows you to choose to display only the simple products or only the variant products. It's pretty useful if you need to quickly find different variant products in different product models, for instance.
```

Comment:
En fait, en sélectionnant "variant", on peut afficher soit les PM uniquement (si l'option "groupée" est activée), soit les variants uniquement (si l'option "dégroupée" est activée). Je mettrais peut être ça après le "product variant display option" du coup.

Reply:
Which feature is the most important one? The most expected? The filter on typology or the display option? @morganen

Reply:
They are both very expected and important... Hard to say, they are complementary.